client/jetpack-connect/schema.js

```diff
@@ -24,6 +24,7 @@ export const authorizeQueryDataSchema = {
 		close_window_after_login: { type: 'string' },
 		from: { type: 'string' },
 		home_url: { type: 'string' },
+		is_popup: { type: 'string' },
```

Comment:
How does this behave with various truth/falsy values in the URL? I see you are interpreting the value as a string, which could cause unintended consequences.

<img width="117" alt="negating-url-params" src="https://user-images.githubusercontent.com/51896/65463039-d7521c80-de0b-11e9-877b-6a161432ce9f.png">

Reply:
good call. I made this more explicit in 11c743240ea18721e466b1931ccb72bf9373c965
and included a unit test for good measure.